concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -49,9 +51,11 @@ def ack_report_computed_task(timestamp = None, subtask_id = None, task_to_comput
         )
 
 
-def task_to_compute(timestamp = None, compute_task_def = None):
+def task_to_compute(timestamp = None, compute_task_def = None, provider_public_key = None, requestor_public_key = None):
     with freeze_time(timestamp):
         return message.tasks.TaskToCompute(
+            provider_public_key = provider_public_key,
```

Comment:
You can add defaults here by doing 
`provider_public_key = provider_public_key if provider_public_key is not None else PROVIDER_PUBLIC_KEY ,`

Then you can remove passing it in all usages, but still override if needed. Same for `requestor_public_key`.

Reply:
Done.